htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/HtmlElement.java

```diff
@@ -252,4 +249,9 @@ public String getCssValue(String name) {
     public String toString() {
         return name;
     }
+
+    @Override
+    public <X> X getScreenshotAs(OutputType<X> outputType) throws WebDriverException {
```

Comment:
PR title says that only Selenium version is updated, but this method is added as well.